app/[locale]/institute/sections/sports/page.tsx

```diff
@@ -13,5 +25,385 @@ export default async function Sports({
+              <TableHead>{text.swimmingPool.manpowerListTitle.name}</TableHead>
+              <TableHead>
+                {text.swimmingPool.manpowerListTitle.quantity}
+              </TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {text.swimmingPool.manpowerList.map((item, index) => (
+              <TableRow key={index}>
+                <TableCell>{index + 1}</TableCell>
+                <TableCell>{item.name}</TableCell>
+                <TableCell>{item.quantity}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+
+        <h5>{text.swimmingPool.maintenance}</h5>
+        <h6>{text.swimmingPool.maintenanceSubTitle}</h6>
+        <Table>
+          <TableHeader>
+            <TableRow>
+              <TableHead>
+                {text.swimmingPool.maintenanceListTitle.sNo}
+              </TableHead>
+              <TableHead>
+                {text.swimmingPool.maintenanceListTitle.name}
+              </TableHead>
+              <TableHead>
+                {text.swimmingPool.maintenanceListTitle.quantity}
+              </TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {text.swimmingPool.maintenanceList.map((item, index) => (
+              <TableRow key={index}>
+                <TableCell>{index + 1}</TableCell>
+                <TableCell>{item.name}</TableCell>
+                <TableCell>{item.quantity}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+
+        <h5>{text.swimmingPool.chemicals}</h5>
+        <h6>{text.swimmingPool.chemicalsSubTitle}</h6>
+        <Table>
+          <TableHeader>
+            <TableRow>
+              <TableHead>{text.swimmingPool.chemicalsListTitle.sNo}</TableHead>
+              <TableHead>
+                {text.swimmingPool.chemicalsListTitle.description}
+              </TableHead>
+              <TableHead>
+                {text.swimmingPool.chemicalsListTitle.quantity}
+              </TableHead>
+              <TableHead>{text.swimmingPool.chemicalsListTitle.unit}</TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {text.swimmingPool.chemicalsList.map((item, index) => (
+              <TableRow key={index}>
+                <TableCell>{index + 1}</TableCell>
+                <TableCell>{item.description}</TableCell>
+                <TableCell>{item.quantity}</TableCell>
+                <TableCell>{item.unit}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+        <h5>{text.swimmingPool.guidelinesTitle}</h5>
+        <ol className="!mt-1 list-inside list-decimal">
+          {text.swimmingPool.guidelines.map((item, index) => (
+            <li
+              key={index}
+              dangerouslySetInnerHTML={{
+                __html: item,
+              }}
+            />
```

Comment:
Don't do this. Instead, initialise MDX to allow for markdown formatting. And convert any `<strong>` tags into `**foo**`.